docs/src/fs_event.rst

```diff
@@ -55,7 +54,6 @@ Data types
             * or kqueue to detect events. This may not work on remote filesystems such
             * as NFS mounts. This flag makes fs_event fall back to calling stat() on a
             * regular interval.
-            * This flag is currently not implemented yet on any backend.
```

Comment:
What was the reason for removing this?  It's still true that they aren't implemented, right?